exercises/practice/resistor-color/resistor_color.asm

```diff
@@ -1,3 +1,7 @@
+default rel
+
+section .data
```

Comment:
If we omit this line from the example solution, so the array is in `section .rodata`,  we see

```
/usr/bin/ld: resistor_color.o: warning: relocation in read-only section `.rodata'    
/usr/bin/ld: warning: creating DT_TEXTREL in a PIE                                   
collect2: error: ld returned 1 exit status                                           
make: *** [Makefile:32: tests] Error 1
```

We can guide students away from experiencing this error.